lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ContainerManager.java

```diff
@@ -202,13 +202,13 @@ public void onNext(final Time value) {
     return normalizedRackNames;
   }
 
-  private void init(final Set<String> rackNames) {
+  private void init() {
```

Comment:
You were right, doesn't even need to pass an argument here. It uses instance variables.